redi-connect-front/src/components/atoms/MediaIcons.tsx

```diff
@@ -5,30 +5,29 @@ import twitter from "../../assets/images/twitter.svg";
 import instagram from "../../assets/images/instagram.svg";
 import medium from "../../assets/images/medium.svg";
 
-const SocialMediaIcons = () => {
-  const icons = [
-    {
-      name: linkedin,
-      link:
-        "https://www.linkedin.com/school/redi-school-of-digital-integration/",
-    },
-    { name: twitter, link: "https://twitter.com/redischool?lang=en" },
-    { name: instagram, link: "https://www.instagram.com/redischool/" },
-    { name: medium, link: "https://www.meetup.com/en-AU/ReDI-school/" },
-  ];
+const icons = [
+  {
+    image: linkedin,
+    link: "https://www.linkedin.com/school/redi-school-of-digital-integration/",
+  },
+  { image: twitter, link: "https://twitter.com/redischool?lang=en" },
+  { image: instagram, link: "https://www.instagram.com/redischool/" },
+  { image: medium, link: "https://www.meetup.com/en-AU/ReDI-school/" },
+];
 
+const SocialMediaIcons = () => {
```

Comment:
Nice to see the change! But I meant to go even further - there's a nice arrow function syntax where you can skip the `{ ... return () }` syntax and jump straight to what is returned:
```
const SocialMediaIcons = () => (
  <div> /* the stuff */ </div>
)
```

:-)

Reply:
Done